requirements-cpu.txt

```diff
@@ -9,7 +9,7 @@ fastapi
 uvicorn[standard]
 pydantic >= 2.0  # Required for OpenAI server.
 prometheus_client >= 0.18.0
-torch == 2.1.2+cpu 
+torch == 2.2.1+cpu 
```

Comment:
Is this a valid pypi package?

Reply:
Not by itself, but in the [documentation](https://docs.vllm.ai/en/latest/getting_started/cpu-installation.html) you are supposed to run this requirements files with an `--extra-index-url` to find the package i.e.
```
pip install -v -r requirements-cpu.txt --extra-index-url https://download.pytorch.org/whl/cpu
```